### ISSUE TITLE
Additional build paths for MSVC projects

### ISSUE THREAD

**Issue by leehu** [Ableton/link#8]
The additional paths needs to be provided which aren't documented:

link\build\modules\ASIOSDK2.3\common 
link\modules\asio-standalone\asio\include 
link\examples\linkaudio  
link\build\modules\ASIOSDK2.3\host 
link\build\modules\ASIOSDK2.3\host\pc

**Comment by fgo-ableton** [comment]
Which target are you referring to? Those should be added when running generating a MSVC project when running cmake.

**Comment by leehu** [comment]
Yeah, I'm not using cmake as i'm already using a makefile generator (JUCE) so I'm looking at integrating Link into already provided MSVC, Xcode and Linux Makefiles. I'm guessing I'm not the only person who isn't going to be using cmake for building as many people will want to incorporate Link into their existing codebase.

As it stands, the page says that if you're not using cmake just add one include directory, set 1 define and you're ready to go, which isn't the correct as all versions need additional include paths for some ASIO includes. Additionally, it says that the build process is simply including the Link.hpp file but if you want access to quantum information it appears as though a number of support classes have to be built into the project aswell.

I totally understand if you guys only want to support cmake builds, but it's going to be confusing if the non-cmake info is not complete. I've never used cmake before so maybe it's possible to run cmake on my JUCE generated makefiles to add all the Link stuff on top - it's something I can investigate...

**Comment by fgo-ableton** [comment]
Thanks for explaining. Of cause asio-standalone has to be in the include paths. I updated that.
I don't understand the problem having to include other files. `Link.hpp` should be the only one required. What do you mean by "access the quantum information"?
